### PR TITLE
Add ESLint to Quasar

### PR DESCRIPTION
**Pull request by allen-tran**
Creates linting JSON file for enforcing linting rules.
Adds github workflows.

### CODE REVIEW

File: printer/PrintHandler.js [hunk 1]
```diff
@@ -4,20 +4,20 @@ const {
...
-const fs = require("fs");
-const s3 = new AWS.S3({ apiVersion: "2012-11-05" });
-var creds = new AWS.Credentials(ACCESS_ID, SECRET_KEY);
-var exec = require("exec");
+} = require('../config/config.json');
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const s3 = new AWS.S3({ apiVersion: '2012-11-05' });
+let creds = new AWS.Credentials(ACCESS_ID, SECRET_KEY);
+let exec = require('exec');
```
Comment (evanugarte) [inline printer/PrintHandler.js]:
```suggestion
const creds = new AWS.Credentials(ACCESS_ID, SECRET_KEY);
const exec = require('exec');
```

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+    - run: npm ci
+    - run: npm run lint
+
```
Comment (evanugarte) [inline .github/workflows/node.js.yml]:
```suggestion

```

**Review by evanugarte — APPROVED** [review]
"Is everything ready?"
"Do it."

image

**Comment by andrewtedi** [comment]
Super cool!

**Review by andrewtedi — APPROVED** [review]

**Pull request merged**